compiler/x/codegen/X86Ops.hpp

```diff
@@ -414,6 +423,11 @@ class TR_X86OpCode
       // OpCodes beyond FENCE are not X86 instructions and hence they do not have binary encodings.
       return op < FENCE ? _binaries[op].encode<TBuffer>(cursor, rex) : cursor;
       }
+   static inline void finalize(TR_X86OpCodes op, uint8_t* cursor)
+      {
+      if (op < FENCE)
```

Comment:
Thanks for clarification. Good point, let me look further into it and make updated.